x-pack/plugins/ml/public/application/datavisualizer/stats_datagrid/components/field_data_expanded_row/number_content.tsx

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, Fragment, ReactNode, useEffect, useState } from 'react';
+import { EuiBasicTable, EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
+
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+import { FieldDataCardProps } from '../../../index_based/components/field_data_card';
+import { kibanaFieldFormat } from '../../../../formatters/kibana_field_format';
+import { numberAsOrdinal } from '../../../../formatters/number_as_ordinal';
+import {
+  MetricDistributionChart,
+  MetricDistributionChartData,
+  buildChartDataFromStats,
+} from '../../../index_based/components/field_data_card/metric_distribution_chart';
+import { TopValues } from '../../../index_based/components/field_data_card/top_values';
+import { ExpandedRowFieldHeader } from '../expanded_row_field_header';
+
+const METRIC_DISTRIBUTION_CHART_WIDTH = 325;
+const METRIC_DISTRIBUTION_CHART_HEIGHT = 210;
+
+interface SummaryTableItem {
+  function: string;
+  display: ReactNode;
+  value: number | string | undefined | null;
+}
+
+export const NumberContent: FC<FieldDataCardProps> = ({ config }) => {
+  const { stats, fieldFormat } = config;
+
+  useEffect(() => {
+    const chartData = buildChartDataFromStats(stats, METRIC_DISTRIBUTION_CHART_WIDTH);
+    setDistributionChartData(chartData);
+  }, []);
+
+  const defaultChartData: MetricDistributionChartData[] = [];
+  const [distributionChartData, setDistributionChartData] = useState(defaultChartData);
+
+  if (stats === undefined) return null;
+  const { min, median, max, distribution } = stats;
+
+  const summaryTableItems = [
+    {
+      function: 'min',
+      display: (
+        <FormattedMessage id="xpack.ml.fieldDataCard.cardNumber.minLabel" defaultMessage="min" />
```

Comment:
The message IDs need editing in this file as they start with `xpack.ml.fieldDataCard.cardNumber` currently.

Reply:
Renamed here https://github.com/elastic/kibana/pull/85726/commits/5eed3fabc69eb50cfa9235e86527c83fd2a14108